modules/swagger-codegen/src/main/resources/pistache-server/api-impl-source.mustache

```diff
@@ -7,7 +7,7 @@
 namespace {{this}} {
 {{/apiNamespaceDeclarations}}
 
-using namespace {{modelNamespace}};
+//using namespace {{modelNamespace}};
```

Comment:
why remove the namespace?

Reply:
I have this issue where no model is used, so no included headerfile, and this line is like a hardcoded namespace declaration that breaks the build in that situation

https://github.com/swagger-api/swagger-codegen/issues/7699

I admit I forgot to remove that in this commit...

Reply:
well, as soon as you fix this one, i can approve it.